EIPS/eip-5069.md

```diff
@@ -0,0 +1,98 @@
+---
+eip: 5069
+title: EIP Editor "Apprentice" Handbook
+description: Formalizes the process for becoming an EIP editor
+author: Pooja Ranjan (@poojaranjan), Pandapip1 (@Pandapip1), et al.
+discussions-to: https://ethereum-magicians.org/t/pr-5069-eip-editor-apprentice-handbook/9137
+status: Living
+type: Informational
+created: 2022-05-02
+requires: 1
+---
+
+## Abstract
+An Ethereum Improvement Proposal (EIP) is a design document providing information to the Ethereum community, or describing a new feature for Ethereum or its processes or environment. The EIP standardization process is a mechanism for proposing new features, for collecting community technical input on an issue, and for documenting the design decisions that have gone into Ethereum. Because improvement proposals are key components of Ethereum blockchain, it is important that they are well reviewed before reaching `final` status. EIPs are stored in text files in a versioned repository which is monitored by the EIP editors.
+
+This EIP describes the process for becoming an EIP editor.
```

Comment:
```suggestion
This EIP describes the recommended process for becoming an EIP editor.
```

Reply:
I would suggest specifying "recommended process" to hope for adoption of the _EIP editor onboarding process_, if pursued as an "Informational" EIP. 

Because as per EIP-1, "Informational EIPs do not necessarily represent Ethereum community consensus or a recommendation, so **users and implementers are free to ignore Informational EIPs or follow their advice**."